portal-web/docroot/html/taglib/ui/form_navigator/lexicon/sections.jspf

```diff
@@ -113,27 +113,26 @@
 				</c:otherwise>
 			</c:choose>
 
-			Liferay.once(
-				'<portlet:namespace />formReady',
-				function(event) {
-					if (!sectionContent.hasClass('in')) {
-						if (focusField.length) {
+			if (focusField.length) {
```

Comment:
I guess what I'm saying is that if focusField has no length, then the panel will not expand in this case.